specification/security/get_settings/SecurityGetSettingsResponse.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { IndexSettings } from '@indices/_types/IndexSettings'
+import { Dictionary } from '@spec_utils/Dictionary'
+
+export class Response {
+  body: {
+    /**
+     * Settings for the index used for most security configuration, including native realm users and roles configured with the API.
+     */
+    security: SecuritySettings
+    /**
+     * Settings for the index used to store profile information.
+     */
+    'security-profile': Dictionary<string, string>
```

Comment:
it's all settings - [server code](https://github.com/elastic/elasticsearch/blob/6f43d898fb5eafd317fabe53a1b6d402b56815f9/x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/action/settings/GetSecuritySettingsAction.java#L70)
```suggestion
    'security-profile': SecuritySettings
```

Reply:
It uses Kebap case here instead of underscores. That's ... interesting? Very interesting API design choice 😅

I have to remember to update my code generator to support that case.